src/cdk-experimental/menu/menu-bar.ts

```diff
@@ -54,60 +49,31 @@ import {MenuAim, MENU_AIM} from './menu-aim';
     {provide: MENU_STACK, useClass: MenuStack},
   ],
 })
-export class CdkMenuBar extends CdkMenuGroup implements Menu, AfterContentInit, OnDestroy {
-  /**
-   * Sets the aria-orientation attribute and determines where menus will be opened.
-   * Does not affect styling/layout.
-   */
-  @Input('cdkMenuBarOrientation') orientation: 'horizontal' | 'vertical' = 'horizontal';
-
-  /** Handles keyboard events for the MenuBar. */
-  private _keyManager: FocusKeyManager<CdkMenuItem>;
-
-  /** Manages items under mouse focus */
-  private _pointerTracker?: PointerFocusTracker<CdkMenuItem>;
+export class CdkMenuBar extends CdkMenuBase implements AfterContentInit, OnDestroy {
+  override orientation: 'horizontal' | 'vertical' = 'horizontal';
```

Comment:
Not a blocker, but what would be a case where there's a vertical menu bar? Maybe we should just support `horizontal`?

Reply:
Yeah I was actually wondering about that, and on the flip side is there any reason to have a `role="menu"` that's horizontal? Should we have `CdkMenu` be always horizontal and `CdkMenuBar` be always vertical?

Reply:
Yeah agreed that a horizontal menu doesn't make sense, I'd be fine if we hardcode the direction. Changing it later will be easy, if we ever need to do it.